docs/src/app/components/pages/components/avatars.jsx

```diff
@@ -38,7 +38,7 @@ export default class AvatarsPage extends React.Component {
           },
           {
             name: 'size',
-            type: 'number',
+            type: 'integer',
```

Comment:
for my usecase more narrow types is better than general types - it doesn't make sense to specify 0.5 pixels for example. If you want to strictly stick with the defined PropTypes i could take those out